### ISSUE TITLE
Implement GEO vCard property

### ISSUE THREAD

**Issue by ciupicri** [owncloud/contacts#879]
There's a `GEO` field starting with vCard 2.1, which is missing in owncloud-7.0.4-3.el6.

Take note that according to [RFC 2426][1] it's something like `GEO:39.95;-75.1667` in vCard 2.1 and 3.0, and according to [RFC 6350][2] it's something like `GEO:geo:39.95,-75.1667` in vCard 4.0.

 [1]: https://tools.ietf.org/html/rfc2426#section-3.4.2
 [2]: https://tools.ietf.org/html/rfc6350#section-5.10

**Comment by babelouest** [comment]
Yes, not all VCard fields are implemented in Owncloud Contacts UI, but if you use an external VCard editor that implements this field and connect it with owncloud through CardDAV, it will be stored in the database as any other field.
